### DISCUSSION TITLE
How to contribute to csv data

### DISCUSSION THREAD

**Discussion by AndryCola** [PokeAPI/pokeapi#857] [Q&A] [answered]
Hi, 
I'm translating all names in Italian for my app and i was wondering if i could share you my data.
Unfortunately i'm not working on CSV files, but i'm using my maria db database.
I think i can export my tables as CSV.

Can someone help me, please?
Thank you

**Comment by Naramsim** [comment]
Hi, I don't think you can export your DB to a CSV ready to be integrated with our CSV files.

https://github.com/PokeAPI/pokeapi/blob/master/data/v2/csv/pokemon_species_names.csv

The best would be for you to create a script that loops your data and fills our CSVs, I think

**Comment by AndryCola** [comment, marked as answer]
Fill your CSV is not a problem, but am i able to push? or i have to create a branch or something like that? 
How does it work?
Thank you

**Reply by Naramsim** [reply]
Ah of course. It's better if you fork this repository so you have your own copy. Then push on whichever branch you want and submit a pull request.

**Comment by AndryCola** [comment]
Ah thank you!
